src/ndsparse.jl

```diff
@@ -483,10 +483,11 @@ internally by many DNDSparse operations.
 """
 function mapchunks(f, t::DNDSparse{K,V}; keeplengths=true) where {K,V}
```

Comment:
nice we should probably remove the V from here.

Reply:
Agreed, I changed the PR accordingly.